internal/ingress/metric/collectors/socket.go

```diff
@@ -30,30 +30,27 @@ import (
 )
 
 type upstream struct {
-	Endpoint       string  `json:"endpoint"`
-	Latency        float64 `json:"upstreamLatency"`
-	ResponseLength float64 `json:"upstreamResponseLength"`
-	ResponseTime   float64 `json:"upstreamResponseTime"`
-	Status         string  `json:"upstreamStatus"`
+	Latency        float64 `json:"uL"`
+	ResponseLength float64 `json:"uResL"`
+	ResponseTime   float64 `json:"uResT"`
+	Status         string  `json:"uS"`
 }
 
 type socketData struct {
```

Comment:
Question: Why do the keys in the payload need to be shortened?

Also to make things more descriptive and shorter than the originals, can you change them to
```
h -> host
s -> stat
n -> ns
i -> ing
s -> svc
p -> loc
```